packager/README.md

```diff
@@ -0,0 +1,43 @@
+Prerequisites:
```

Comment:
Consider markdown styling, headings or whatnot.

Reply:
Tweaked the README a bit.